library/VTKExtensions/Rendering/vtkF3DRendererWithColoring.cxx

```diff
@@ -463,15 +574,10 @@ void vtkF3DRendererWithColoring::ConfigureVolumeForColoring(vtkSmartVolumeMapper
 }
 
 //----------------------------------------------------------------------------
-void vtkF3DRendererWithColoring::ConfigureScalarBarActorForColoring(
-  vtkScalarBarActor* scalarBar, vtkDataArray* array, int component, vtkColorTransferFunction* ctf)
+void vtkF3DRendererWithColoring::ConfigureScalarBarActorForColoring(vtkScalarBarActor* scalarBar,
+  const std::string& arrayName, int component, vtkColorTransferFunction* ctf)
 {
-  if (!array)
-  {
-    return;
-  }
-
-  std::string title = array->GetName();
+  std::string title = arrayName;
```

Comment:
you can change the function to take a string copy and avoid this local variable.